src/renderer/app/components/SvgIcon/index.ts

```diff
@@ -0,0 +1,4 @@
+import SvgIcon from "./SvgIcon.vue";
+
+export default SvgIcon;
```

Comment:
Avoid default exports due to them being bad practice because they could be imported differently. Always require an explicit export and import unless you are forced to provide a default export by some tooling like Vue plugins.